Cooper/ClientApp/src/app/guards/auth-guard.ts

```diff
@@ -1,20 +1,19 @@
-import { JwtHelperService } from '@auth0/angular-jwt';
 import { Injectable } from '@angular/core';
 import { Router, CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
 
 @Injectable()
 export class AuthGuard implements CanActivate {
-  constructor(private jwtHelper: JwtHelperService, private router: Router) {
+  constructor(private router: Router) {
   }
   canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot) {
-    var token = localStorage.getItem("jwt");
+    var token = localStorage.getItem("JwtCooper");
 
-    if (token && !this.jwtHelper.isTokenExpired(token)){
-      console.log(this.jwtHelper.decodeToken(token));
+    if (token != null)
```

Comment:
Don't compare anything for null.
String from storage can be in 4 states: undefined, null, '' (empty string), 'some_text'. First 3 cases should return false.
So always use if (token) {.....}. It will return false if value is null, undefined, zero(for numbers), empty string, ...